index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Bimbala Desktop</title>
+    <!-- Include the Tailwind CSS stylesheet -->
+    <link href="https://cdn.jsdelivr.net/npm/tailwindcss@2.2.15/dist/tailwind.min.css" rel="stylesheet">
+</head>
+
+<body>
+    <section class="flex h-screen flex-col items-center md:flex-row">
+        <div class="hidden h-screen w-full bg-indigo-600 md:w-1/2 lg:block xl:w-2/3">
+            <img src="https://source.unsplash.com/random" alt="" class="h-full w-full object-cover" />
```

Comment:
Let's narrow the range of photos to buildings only. They remind of tenants and multitenancy.
```suggestion
            <img src="https://source.unsplash.com/random/?building" alt="" class="h-full w-full object-cover" />
```